sound/soc/sof/ipc.c

```diff
@@ -25,18 +25,6 @@ static void ipc_stream_message(struct snd_sof_dev *sdev, u32 msg_cmd);
  * IPC message Tx/Rx message handling.
  */
 
-/* SOF generic IPC data */
```

Comment:
IPC in caps in commit message please

Reply:
if this is the first patch in the series, there's no intro to IPC4, maybe you could simply say: "Move definition of struct snd_sof_ipc to the header file so it can be shared with new IPC versions"

Reply:
> if this is the first patch in the series, there's no intro to IPC4, maybe you could simply say: "Move definition of struct snd_sof_ipc to the header file so it can be shared with new IPC versions"

Done. Thanks